solutions/cf-702C.mdx

```diff
@@ -188,6 +188,73 @@ int main() {
 
 </CPPSection>
 	
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class CellularNetwork2 {
+	static final int MOD = 1000000007;
+	static final int MAXIMUM_SIZE = 100000;
+
+	public static void main (String[] args) {
+		Kattio io = new Kattio();
+
+		int n = io.nextInt();
+		int m = io.nextInt();
+		int[] dist = new int[MAXIMUM_SIZE];
+
+		Arrays.fill(dist, 2 * MOD);
+
+		int[] a = new int[n];
+		int[] b = new int[m];
+
+		for (int x = 0; x < n; x++) {
```

Comment:
Why not i

Reply:
bro

Reply:
what

Reply:
you used i as the for loop variable everywhere else

Reply:
yeah be consistent lol